docs/general/transaction-attacks.md

```diff
@@ -60,6 +60,22 @@ This will be showed in the Signer app and a careful user will notice it. If the
 Signer is already incorrect (or the Signer is corrupted) there is the risk of signing a non-intended
 transaction without the possibility of verifying it.
 
+### Replay Attack
+
+A replay attack is where past transactions can be replayed (same [balance](#balance-transfers),
+receiver account, etc.) without knowing private keys. This could happen in the context of
+[reaping accounts](./learn-accounts.md#existential-deposit-and-reaping) because the reaping process
+resets the nonce value. If all signed transactions until the nonce before the reaping event were
+immortal, all past transactions can be replayed once the account is refunded. There is no need for
+the attacker to know your private key, valid signatures for those past transactions and nonces
+already exist and are stored on-chain (meaning the private key was already used to generate those
+signatures).
+
+Making a [transaction mortal](../learn/learn-transactions.md#mortal-and-immortal-extrinsics) will
+almost certainly ensure that replay attacks are not possible, with the only exception being if the
+account is reaped and then re-funded shortly after submitting a mortal transaction, and then an
+attacker replays that transaction within the mortality window (i.e., the specified block interval).
```

Comment:
... AND that transaction which is valid (e.g. if you only have 50 DOT, and you try to issue an extrinsic sending 100 DOT, the transfer will fail)